src/main/java/it/pagopa/pn/delivery/svc/NotificationAttachmentService.java

```diff
@@ -170,15 +171,14 @@ private NotificationAttachmentDownloadMetadataResponse downloadDocumentWithRedir
 
             FileInfos fileInfos = computeFileInfo( fileDownloadIdentify, notification );
 
-            assert fileInfos.fileDownloadResponse.getDownload() != null;
-            assert fileInfos.fileDownloadResponse.getDownload().getRetryAfter() != null;
+            BigDecimal retryAfter = fileInfos.fileDownloadResponse.getDownload().getRetryAfter();
             return NotificationAttachmentDownloadMetadataResponse.builder()
                     .filename( fileInfos.fileName)
                     .url( fileInfos.fileDownloadResponse.getDownload().getUrl() )
-                    .contentLength(fileInfos.fileDownloadResponse.getContentLength().intValue())
+                    .contentLength( fileInfos.fileDownloadResponse.getContentLength().intValue() )
```

Comment:
Sicuro al 100% che qui non si vada mai in NPE ?

Reply:
fixata